src/bci_build/package.py

```diff
@@ -854,13 +854,17 @@ def description(self) -> str:
         :py:attr:`BaseContainerImage.pretty_name` to generate a description.
 
         """
+
+        description_formatters = {
```

Comment:
Please mention this in the docstring of the description. Also, this will certainly mess with `f""` strings, so I'm not overly happy with this change :-/